### DISCUSSION TITLE
Introducing a new options search 🔍✨

### DISCUSSION THREAD

**Discussion by getchoo** [catppuccin/nix#391] [Announcements]
With v1.2 on the way and v2.0 hopefully not far off either, we've been wanting a way to handle multiple versions of our options documentation on [nix.catppuccin.com](https://nix.catppuccin.com/), as well as make it easier to find the options you're looking for. So after [a bit of work](https://github.com/catppuccin/nix/pull/390), we are proud to announce new pages for exploring our modules!

These can be found at [nix.catppuccin.com/options](https://nix.catppuccin.com/options), where search engines (powered by the amazing [NuschtOS/search](https://github.com/NuschtOS/search)) for each minor release and our "rolling" main branch are listed. Our hope is that this will make things much more discoverable, and introduce a workflow similar to that of search.nixos.org

We would love to get feedback on this, along with the rest of our documentation :3

**Comment by sgoudham** [comment]
Nice job! I think we could also raise a pull request on [NuschtOS/search](https://github.com/NuschtOS/search) listing `nix.catppuccin.com/search/rolling` as [another example of a deployment](https://github.com/NuschtOS/search?tab=readme-ov-file#deployments).

Sidenote, it would be good to redirect `nix.catppuccin.com/search` to the latest stable but I'm not sure if this is possible since mdbook is hijacking that at the minute 🤔 

Update: Looks like mdBook does [support redirects](https://rust-lang.github.io/mdBook/format/configuration/renderers.html#outputhtmlredirect) so this should allow us to do it, as long as we (manually or automatically) update the value to redirect to the latest stable tag at the time of building.

**Reply by getchoo** [reply]
> Nice job! I think we could also raise a pull request on [NuschtOS/search](https://github.com/NuschtOS/search) listing nix.catppuccin.com/search/rolling as [another example of a deployment](https://github.com/NuschtOS/search?tab=readme-ov-file#deployments).

Opened https://github.com/NuschtOS/search/pull/180 :)

> Sidenote, it would be good to redirect nix.catppuccin.com/search to the latest stable but I'm not sure if this is possible since mdbook is hijacking that at the minute 🤔

Because of some mdbook/Pages/our build process shenanigans, best we can can have is https://nix.catppuccin.com/search/index.html and https://nix.catppuccin.com/search/stable.html
